webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const TerserPlugin = require('terser-webpack-plugin');
+
+const path = require('path');
+const package = require('./package.json');
+
+module.exports = {
+  devtool: "nosources-source-map",
+  entry: './src/lob-address-elements.js',
+  optimization: {
+    minimizer: [
+      new TerserPlugin({
+        parallel: true,
+        terserOptions: {
+          sourceMap: true
+        },
+      })
+    ]
+  },
+  output: {
```

Comment:
since we're going to continue to use jq and algolia for this next release, I'd like to get your thoughts on what would be involved with packaging these source files and requiring those as npm modules (I'm unsure if the algolia plugin has a packagable version, but it might be easier for dependency management).

Reply:
I think I found the right [algolia module](https://www.npmjs.com/package/algoliasearch).  If we switch to using npm modules they will be included in our minified file. I think our current method for conditionally installing these dependencies is the best for keeping our build small.